### PR TITLE
fix: use main color for local style source

### PR DESCRIPTION
**Pull request by TrySound**
Fixes https://github.com/webstudio-is/webstudio/issues/2610

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview
- [ ] hi @taylornowotny, I need you to do
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by kof — APPROVED** [review]

**Comment by kof** [comment]
Didn't test, trusting you on this

**Comment by kof** [comment]
@taylornowotny this PR waits for your ✅

Was the description clear about the change and what to test?

**Review by taylornowotny — APPROVED** [review]
Sweet, looks great 👍

Can we possibly change the min width of style sources from 40px to 32px? Not critical, it would just look better. Thanks

**Comment by taylornowotny** [comment]
I think you'd agree @kof the min width feels too large right now

Screenshot 2023-11-20 at 2 53 14 PM

**Comment by kof** [comment]
I was just testing and got surprised, when did we change the colors this way?

image

File: apps/builder/app/builder/features/style-panel/style-source/style-source.tsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ const EditableText = ({
 const StyleSourceContainer = styled(Box, {
   display: "inline-flex",
   borderRadius: theme.borderRadius[3],
-  minWidth: theme.spacing[13],
+  minWidth: theme.spacing[15],
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/style-source/style-source.tsx:154]: this seems big enough
```suggestion
  minWidth: theme.spacing[14],
```

**Comment by kof** [comment]
@taylornowotny yes agree on 36

**Pull request merged**